pkg/database/realm.go

```diff
@@ -293,10 +293,19 @@ func (r *Realm) GetLongCodeDurationHours() int {
 }
 
 // BuildSMSText replaces certain strings with the right values.
-func (r *Realm) BuildSMSText(code, longCode string) string {
+func (r *Realm) BuildSMSText(code, longCode string, enxDomain string) string {
 	text := r.SMSTextTemplate
 
-	text = strings.ReplaceAll(text, SMSENExpressLink, fmt.Sprintf("ens://v?r=%s&c=%s", SMSRegion, SMSLongCode))
+	if enxDomain == "" {
+		// preserves legacy behavior.
+		text = strings.ReplaceAll(text, SMSENExpressLink, fmt.Sprintf("ens://v?r=%s&c=%s", SMSRegion, SMSLongCode))
+	} else {
+		text = strings.ReplaceAll(text, SMSENExpressLink,
+			fmt.Sprintf("https://%s.%s/v?c=%s",
+				strings.ToLower(r.RegionCode),
```

Comment:
I don't think we currently enforce uniqueness on region codes right now, just realm names. Should we add a database constraint for that?

Reply:
we can file an issue for that. I'm not confident it would help anything since it's only specific to one sever